.vsts-dotnet.yml

```diff
@@ -91,7 +91,27 @@ stages:
                 /p:TeamName=MSBuild
                 /p:DotNetPublishUsingPipelines=true
       displayName: Build
-      condition: succeeded()
+      condition: and(succeeded(), not(startsWith(variables['Build.SourceBranch'], 'refs/heads/exp/')))
```

Comment:
Is it possible to define a new variable with this condition and use that in a single invocation of `cibuild.cmd`?

Reply:
Livar just pointed me to an example for that. Much cleaner!